io.github.maurycyliebner.enve.yaml

```diff
@@ -0,0 +1,186 @@
+app-id: io.github.maurycyliebner.enve
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: enve
+rename-desktop-file: enve.desktop
+rename-icon: enve
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
Can it use just `home`?

Reply:
I think I have seen a comment from someone (maybe @TingPing), that from a security point, it doesn't make a big difference, but I can change it (I'm thought, you need it for external disks , but I think i have to add another permission for that).

Reply:
Kinda yes, but it's always just slightly better.